.github/ISSUE_TEMPLATE/1-lts-release-checklist.md

```diff
@@ -38,7 +38,7 @@ This role should rotate between LTS releases
 
 - [ ] Backporting announcement email - [generate-backporting-announcement script](https://github.com/jenkins-infra/backend-commit-history-parser/blob/master/bin/generate-backporting-announcement)
 
-- [ ] Update jira labels with the selected issues, e.g. `2.277.2-fixed`, `2.277.2-rejected`, and remove `lts-candidate`
+- [ ] Update jira labels for [lts-candidate issues](https://issues.jenkins.io/issues/?filter=12146), either add `2.277.2-fixed` and remove `lts-candidate` or add `2.277.2-rejected`, and retain `lts-candidate`
```

Comment:
Better, you can remove the lts candidate when rejecting though if it just doesn’t make sense

Reply:
It's an unusual edge case, I think this is fine. We'd also expect a comment explaining why it's not suitable for backporting etc.; just document the default for folks who'd otherwise be unsure how this works to make it easier for them.